Patch for #2

kartikarora opened the pull request:
Updated gradle plugin and dependencies
Updated .gitignore
Improved UI to follow material design guidelines

raj-subhankar commented:
@kartikarora hi can you add screenshots? Also please git pull once before sending a pull request.

kartikarora committed:
Patch for #2

Updated gradle plugin and dependencies
Updated .gitignore
Improved UI to follow material design guidelines

kartikarora commented:
@raj-subhankar Rebased with your commit! Here's the screenshot!

Do test the branch out because I've made some UX changes which cannot be shown in the screenshot!

![device-2016-10-19-000152](https://cloud.githubusercontent.com/assets/6591617/19491211/bfd70812-958f-11e6-9286-15d99a0780c4.png)

raj-subhankar commented:
@kartikarora Good work! Thanks
